coderd/users_test.go

```diff
@@ -1521,10 +1560,8 @@ func TestUsersFilter(t *testing.T) {
 		FilterF func(f codersdk.UsersRequest, user codersdk.User) bool
 	}{
 		{
-			Name: "All",
-			Filter: codersdk.UsersRequest{
-				Status: codersdk.UserStatusSuspended + "," + codersdk.UserStatusActive,
-			},
+			Name:   "All",
+			Filter: codersdk.UsersRequest{},
```

Comment:
updated this since "All" filter should include all the users

Reply:
Given this is done twice, I assume this was intentional. Just a sanity check that all users are either suspended or active. I think we should just keep the original.